src/DatePicker/index.js

```diff
@@ -68,7 +70,7 @@ export default class DatePicker {
         <button type="button" class="date-picker__toggle" aria-label="Toggle Date Picker">Toggle Date Picker</button>
```

Comment:
Should this toggle be associated with the datepicker it toggles in some way with `aria-roles`?